src/View/Helper/HtmlHelper.php

```diff
@@ -784,6 +799,11 @@ public function getCrumbList(array $options = [], $startText = false)
      */
     protected function _prepareCrumbs($startText, $escape = true)
     {
+        deprecationWarning(
+            'HtmlHelper::_prepareCrumbs() is deprecated. ' .
```

Comment:
even for `protected` methods ?

Reply:
Well, you can call them? @markstory has not told me any restrictions and I don't think it hurts to have it in one more place?

Reply:
Makes sense to deprecate those as well, as long as it is not `final private` ;-)...

Reply:
>Well, you can call them when extending the class
OK :+1:

Reply:
We should warn on protected methods as well. They will be removed and break code in the future.